alembic/versions/9965c6e1d607_2024_06_06-created_company_table.py

```diff
@@ -0,0 +1,61 @@
+"""created_company_table
+
+Revision ID: 9965c6e1d607
+Revises: f3eb3f84334b
+Create Date: 2024-06-06 10:06:35.875369
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "9965c6e1d607"
+down_revision: Union[str, None] = "f3eb3f84334b"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "companies",
+        sa.Column("id", sa.Integer, primary_key=True, autoincrement=True),
+        sa.Column("name", sa.String, nullable=False),
+        sa.Column("registration_number", sa.String, unique=True, nullable=False),
```

Comment:
Research the length of company registration numbers

Reply:
They don't have a limit